src/main/webapp/resources/js/utilities/link-utilities.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
```

Comment:
File comment

Reply:
Updated in ae0c96c5abfdd9cd32d193785314afd51f544a1e